utils/tang/files/tang.init

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+USE_PROCD=1
+
+. $IPKG_INSTROOT/lib/functions/network.sh
+
+start_service() {
+	KEYS=$(find /usr/share/tang/db -name "*.jw*" -maxdepth 1 | wc -l)
+	if [ -z "${KEYS}" ] || [ "${KEYS}" = "0" ]; then # if db is empty generate new key pair
```

Comment:
```suggestion
	if [ -z "$KEYS" ] || [ "$KEYS" = 0 ]; then # if db is empty generate new key pair
```

Reply:
I see it, but I do not see the reason for this change. Could you elaborate?

Reply:
personal preference, mostly

Reply:
Thanks. Unless there is a recommended standard, I prefer to leave them as is.